install-tests/react-native/install.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+react_native_version="${1:-nightly}"
```

Comment:
I am getting an error when trying to install with the default arguments.  It seems the `nightly` version of RN breaks the peer dependency `react-native >= 0.60.0`

Reply:
I just got a compile error with `nightly` as well, `next` is probably a better default option.

Reply:
It's because the nightly version is 0.0.x.  We could just use a `--force` flag

Reply:
I think `next` is better, it's a little less brittle but we'll still gain an edge compared to end-users installing from `latest.

Reply:
I agree.  `next` hits the RCs for React Native, so that has a lot of value.  `nightly` may just be chaotic and not be as useful.